packages/replay-internal/src/session/index.ts

```diff
@@ -1 +0,0 @@
-export * from './createSession';
```

Comment:
`createSession` is just imported directly so this just removes in barrel/index file